LeanKit-Labs/hyped-swagger-middleware
# Added CORS headers to swagger endpoint

This adds CORS headers to the swagger endpoint. I also updated the development dependencies as well as the lodash version. This removes all production dependency vulnerabilities from this project.

Testing Steps:

1. `npm link` into the host project
2. Attempt to use http://petstore.swagger.io/?url=https://DOMAIN/io/swagger/1/swagger.json replacing `DOMAIN` with the test domain
3. It should load and not throw a CORS error
